src/Compilers/VisualBasic/Portable/Symbols/Source/SourceNamedTypeSymbol.vb

```diff
@@ -2559,6 +2600,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                         ImmutableArray.Create(New TypedConstant(compilation.GetWellKnownType(WellKnownType.System_Type), TypedConstantKind.Type, originalType)),
                         isOptionalUse:=True))
             End If
+
+            If Me.IsMicrosoftCodeAnalysisEmbeddedAttribute() Then
+                Dim earlyAttributeData = GetEarlyDecodedWellKnownAttributeData()
+
+                If earlyAttributeData Is Nothing OrElse Not earlyAttributeData.HasCodeAnalysisEmbeddedAttribute Then
```

Comment:
>earlyAttributeData Is Nothing OrElse Not earlyAttributeData.HasCodeAnalysisEmbeddedAttribute [](http://example.com/codeflow?start=19&length=92)

Are we testing positive and negative effect of this condition?

Reply:
Yes, similar to C#. I will also add a similar additional test for another early well-known attribute to test the `OrElse` bit.